### PR TITLE
Fix suspend contracts bug

### PR DESCRIPTION
**Pull request by progsmile**
#3541

### CODE REVIEW

File: apps/core/lib/core/contracts/contracts_suspender.ex [hunk 1]
```diff
@@ -1,28 +1,47 @@
...
   end
 
-  def maybe_suspend_contracts?(%{changes: changes}, :legal_entity) do
-    maybe_suspend_contracts?(changes, ~w(name addresses status)a)
+  def should_suspend_contracts?(%{changes: changes}, :legal_entity) do
+    should_suspend_contracts?(changes, ~w(name addresses status)a)
   end
 
-  def maybe_suspend_contracts?(changes, keys) when is_list(keys) do
+  def should_suspend_contracts?(changes, keys) when is_list(keys) do
```
[the remark this answers is not in the document]
Comment (progsmile) [reply apps/core/lib/core/contracts/contracts_suspender.ex]: Suppose `should` prefix is better for functions that return bool
We have `action based` functions with prefix `maybe`, e.g: `maybe_send_sms`, `maybe_terminate_person_declaration`
So it's mistaken

[the remark this answers is not in the document]
Comment (kiurchv) [reply apps/core/lib/core/contracts/contracts_suspender.ex]: IMO `suspend_contracts?` is enough for naming that function

[the remark this answers is not in the document]
Comment (progsmile) [reply apps/core/lib/core/contracts/contracts_suspender.ex]: yep, nice choice 
@PavelVesnin ?

[the remark this answers is not in the document]
Comment (PavelVesnin) [reply apps/core/lib/core/contracts/contracts_suspender.ex]: by convention functions with question mark on the end should return boolean value. So I'm agree

**Pull request merged**